src/toM3u8.js

```diff
@@ -291,10 +291,17 @@ export const toM3u8 = (dashPlaylists, locations, sidxMapping = {}) => {
 
```

Comment:
We have to add the audio group playlists to our big list of playlists so that when we are audio only they can be used as the main playlist.